app/pb_kits/playbook/pb_label_value/label_value.rb

```diff
@@ -2,55 +2,18 @@
 
 module Playbook
   module PbLabelValue
-    class LabelValue < Playbook::PbKit::Base
-      PROPS = %i[configured_classname
-                 configured_data
-                 configured_id
-                 configured_label
-                 configured_value].freeze
+    class LabelValue
+      include Playbook::Props
 
-      def initialize(classname: default_configuration,
-                     data: default_configuration,
-                     id: default_configuration,
-                     label: default_configuration,
-                     value: default_configuration)
-        self.configured_classname = classname
-        self.configured_data = data
-        self.configured_id = id
-        self.configured_label = label
-        self.configured_value = value
-      end
-
-      def label
-        if is_set? configured_label
-          pb_label = Playbook::PbCaption::Caption.new(text: configured_label)
-          ApplicationController.renderer.render(partial: pb_label, as: :object)
-        end
-      end
-
-      def value
-        pb_body = Playbook::PbBody::Body.new do
-          default_value(configured_value, "")
-        end
-        ApplicationController.renderer.render(partial: pb_body, as: :object)
-      end
-
-      def kit_class
-        "pb_label_value_kit"
-      end
-
-      def to_partial_path
-        "pb_label_value/label_value"
-      end
+      partial "pb_label_value/label_value"
 
-    private
+      prop :label, type: Playbook::Props::String
+      prop :value, type: Playbook::Props::String,
+                   default: ""
```

Comment:
So for the props, the default type is `string` & the default value in `nil`, so the `, type: Playbook::Props::String` & `, defaults: ""` wouldn't be necessary here.

Reply:
Thank you! Changing that here / in the other PRs too

Reply:
I was about to add the same comment to the other PR... I shall refrain. lol

Reply:
Haha sounds good - thanks!